src/pipelines/tailwind_css.rs

```diff
@@ -95,17 +99,28 @@ impl TailwindCss {
 
         let mut args = vec!["--input", &path_str, "--output", &file_path];
 
+        if let Some(tailwind_config) = self.tailwind_config.as_ref() {
+            args.push("--config");
+            args.push(tailwind_config);
+        }
+
         if self.cfg.minify_asset(self.no_minify) {
             args.push("--minify");
         }
 
         let rel_path = common::strip_prefix(&self.asset.path);
         tracing::debug!(path = ?rel_path, "compiling tailwind css");
+
+        let current_dir = if cfg!(target_os = "windows") {
```

Comment:
I am wondering if it would make more sense to have `self.cfg.core.working_directory` to always be processes with `dunce::simplified`? The docs say, on non-windows it's a no-op anyway. And this might be required for other locations in trunk as well.

Reply:
ah yeap; one moment -- it took like 20 jumps to find that it's initialized in `src\config\rt\core.rs` 😅